### ISSUE TITLE
Popup blocked by browser

### ISSUE THREAD

**Issue by markgdawson** [atlas-engineer/nyxt#882]
**Describe the bug**

Clicking on the "Continue with google" button, here:

https://account.samsung.com/accounts/v1/mysamsung-en-gb/signInGate?response_type=code&client_id=mvz7wwl2m0&locale=en-GB&countryCode=GB&redirect_uri=https:%2F%2Fapi.my-samsung.com%2Fsa%2Fpromotool%2Fv1%2Floggedin&state=en-gb~uk~en-GB~FRHZMEYBVUHMZ~1596571980654&goBackURL=https:%2F%2Fmy-samsung.com%2Fuk%2Fmysamsung%2Foffers%2F#%2FFRHZMEYBVUHMZ%2Fnot-logged-in

gives the following error (when the web inspector is open):
```
[Error] Failed to load resource: the server responded with a status of 404 (Not Found) (angular-messages.min.js.map, line 0)
[Log] response.error popup_blocked_by_browser (signIn.js, line 333)
[Error] The source list for Content Security Policy directive 'script-src' contains an invalid source: ''strict-dynamic''. It will be ignored.
```

**Information**
Next Version: 1.5.0-2a98df1e 
Lisp Implementation: SBCL 
Lisp Version: 1.5.8.71-176d4df3b 
Operating System: Linux 5.1.15-surface-linux-surface 
Features: (WEBKIT2 WEBKIT2-2.28 WEBKIT2-EMOJI WEBKIT2-MEDIA WEBKIT2-SANDBOXING
           GTK-3-22 GTK-3-20 GTK-3-18 GTK-3-16 GTK-3-14 GTK-3-12 GTK-3-10
           GTK-3-8 GTK-3-6 GTK-3-4 GTK GDK-3-22 GDK-3-20 GDK-3-18 GDK-3-16
           GDK-3-14 GDK-3-12 GDK-3-10 GDK-3-8 GDK-3-6 GDK-3-4 CAIRO-1-10
           CAIRO-1-12 GDK-PIXBUF GLIB-2-30 GLIB-2-32 GLIB-2-34 GLIB-2-36
           GLIB-2-38 GLIB-2-40 GLIB-2-42 GLIB-2-44 GLIB-2-46 GLIB-2-48
           GLIB-2-50 GLIB-2-52 GLIB-2-54 GLIB-2-56 GLIB-2-58 GLIB
           FSET-EXT-STRINGS SWANK PLUMP-UTF-32 GLOBAL-VARS DECLARE-TYPES
           PARENSCRIPT NAMED-READTABLES OSICAT-FD-STREAMS LPARALLEL 21BIT-CHARS
           CL-FAD CHUNGA FLEXI-STREAMS CLOSER-MOP CL-PPCRE-UNICODE CL-UNICODE
           CL-PPCRE BORDEAUX-THREADS LPARALLEL.WITH-CLTL2 LPARALLEL.WITH-CAS
           LPARALLEL.WITH-STEALING-SCHEDULER SPLIT-SEQUENCE GRAY-STREAMS
           FLAT-NAMESPACE X86-64 UNIX CFFI FLAT-NAMESPACE SEQUENCE-EMPTYP
           FAST-IO-SV FAST-IO SBCL-USES-SB-ROTATE-BYTE ASDF-SYSTEM-CONNECTIONS
           CL-JSON-CLOS CL-JSON THREAD-SUPPORT QUICKLISP
           SB-BSD-SOCKETS-ADDRINFO ASDF3.3 ASDF3.2 ASDF3.1 ASDF3 ASDF2 ASDF
           OS-UNIX NON-BASE-CHARS-EXIST-P ASDF-UNICODE X86-64 64-BIT
           ALIEN-CALLBACKS ANSI-CL AVX2 C-STACK-IS-CONTROL-STACK CALL-SYMBOL
           COMMON-LISP COMPACT-INSTANCE-HEADER COMPARE-AND-SWAP-VOPS
           CYCLE-COUNTER ELF FP-AND-PC-STANDARD-SAVE GENCGC IEEE-FLOATING-POINT
           IMMOBILE-CODE IMMOBILE-SPACE INTEGER-EQL-VOP LARGEFILE LINKAGE-TABLE
           LINUX LITTLE-ENDIAN OS-PROVIDES-BLKSIZE-T OS-PROVIDES-DLADDR
           OS-PROVIDES-DLOPEN OS-PROVIDES-GETPROTOBY-R OS-PROVIDES-POLL
           OS-PROVIDES-PUTWC OS-PROVIDES-SUSECONDS-T PACKAGE-LOCAL-NICKNAMES
           RELOCATABLE-HEAP SB-CORE-COMPRESSION SB-DOC SB-EVAL SB-FUTEX SB-LDB
           SB-PACKAGE-LOCKS SB-SIMD-PACK SB-SIMD-PACK-256 SB-SOURCE-LOCATIONS
           SB-THREAD SB-UNICODE SB-XREF-FOR-INTERNALS SBCL
           STACK-ALLOCATABLE-CLOSURES STACK-ALLOCATABLE-FIXED-OBJECTS
           STACK-ALLOCATABLE-LISTS STACK-ALLOCATABLE-VECTORS
           STACK-GROWS-DOWNWARD-NOT-UPWARD UNDEFINED-FUN-RESTARTS UNIX
           UNWIND-TO-FRAME-AND-CALL-VOP)

**Comment by Ambrevar** [comment]
I believe this is a duplicate of #768.

**Comment by jmercouris** [comment]
This issue has been fixed according to #768 , please reopen if you still have issues!
